app/controllers/students_controller.rb

```diff
@@ -30,6 +29,11 @@ def show
     }
   end
 
+  def restricted_notes
+    raise Exceptions::EducatorNotAuthorized unless current_educator.can_view_restricted_notes
+    show(restricted_notes: true)
```

Comment:
Yeah, agreed. I think having `restricted_notes` as a parameter on `student_feed` is sufficient without having to thread it all the way through.